librz/main/rz-bin.c

```diff
@@ -235,19 +235,20 @@ static int rzbin_show_help(int v) {
 	}
 	if (v) {
 		printf("Environment:\n"
-		       " RZ_NOPLUGINS:                                         # do not load shared plugins (speedup loading)\n"
-		       " RZ_BIN_LANG:      e bin.lang                          # assume lang for demangling\n"
-		       " RZ_BIN_DEMANGLE=0:e bin.demangle                      # do not demangle symbols\n"
-		       " RZ_BIN_MAXSTRBUF: e str.search.buffer_size            # specify maximum buffer size\n"
-		       " RZ_BIN_STRFILTER: e bin.str.filter                    # rizin -qc 'e bin.str.filter=?"
+		       " RZ_BIN_CODESIGN_VERBOSE:                               # make code signatures verbose\n"
+		       " RZ_BIN_DEBASE64:         e bin.debase64                # try to debase64 all strings\n"
+		       " RZ_BIN_DEBUGINFOD_URLS:  e bin.dbginfo.debuginfod_urls # use alternative debuginfod server\n"
+		       " RZ_BIN_DEMANGLE=0:       e bin.demangle                # do not demangle symbols\n"
+		       " RZ_BIN_LANG:             e bin.lang                    # assume lang for demangling\n"
+		       " RZ_BIN_MAXSTRBUF:        e str.search.buffer_size      # specify maximum buffer size\n"
+		       " RZ_BIN_PDBSERVER:        e pdb.server                  # use alternative PDB server\n"
+		       " RZ_BIN_PREFIX:           e bin.prefix                  # prefix symbols/sections/relocs with a specific string\n"
+		       " RZ_BIN_STRFILTER:        e bin.str.filter              # rizin -qc 'e bin.str.filter=?"
 		       "?' -\n"
-		       " RZ_BIN_STRPURGE:  e bin.str.purge                     # try to purge false positives\n"
-		       " RZ_BIN_DEBASE64:  e bin.debase64                      # try to debase64 all strings\n"
-		       " RZ_BIN_PDBSERVER: e pdb.server                        # use alternative PDB server\n"
-		       " RZ_BIN_SYMSTORE:  e pdb.symstore                      # path to downstream symbol store\n"
-		       " RZ_BIN_PREFIX:    e bin.prefix                        # prefix symbols/sections/relocs with a specific string\n"
-		       " RZ_BIN_DEBUGINFOD_URLS: e bin.dbginfo.debuginfod_urls # use alternative debuginfod server\n"
-		       " RZ_CONFIG:                                            # sdb config file\n");
+		       " RZ_BIN_STRPURGE:         e bin.str.purge               # try to purge false positives\n"
+		       " RZ_BIN_SYMSTORE:         e pdb.symstore                # path to downstream PDB symbol store\n"
+		       " RZ_CONFIG:                                             # config file\n"
+		       " RZ_NOPLUGINS:                                          # do not load shared plugins (speedup loading)\n");
```

Comment:
```suggestion
		       " RZ_NOPLUGINS:                                          # do not load plugins\n");
```

Reply:
Thanks for the reviews, I need to go to church, I will fix it as soon as I come back.